warehouse/warehouse.go

```diff
@@ -1679,7 +1679,7 @@ func Start(ctx context.Context, app app.Interface) error {
 
 	runningMode := config.GetEnv("RSERVER_WAREHOUSE_RUNNING_MODE", "")
 	if runningMode == DegradedMode {
-		pkgLogger.Infof("WH: Running warehouse service in degared mode...")
+		pkgLogger.Infof("WH: Running warehouse service in degraded mode...")
```

Comment:
@chandumlg is it safe to change existing log statements? I don't see why it shouldn't be unless we are performing any kind of log processing, e.g. for alerting or reporting purposes that relies on the existing log formats.